internal/apiclient/fftm_client.go

```diff
@@ -19,7 +19,7 @@ package apiclient
 import (
 	"context"
 
-	"github.com/go-resty/resty/v2"
+	resty "github.com/go-resty/resty/v2"
```

Comment:
Was there a reason for needing to qualify here, and in the other files e.g. `pkg/apitypes/ulid.go`?

Reply:
So with Go 1.23 it was complaining that it couldn't find that package, so I think there was a change to the import name. Normally it's the name of the package but it might be taking v2 now